R/rd-html.R

```diff
@@ -100,8 +100,8 @@ as_html.tag_subsection <- function(x, ..., subsection_level = 3L) {
   h <- paste0("h", subsection_level)
 
   paste0(
-    "<", h, " class='hasAnchor' id='arguments'>",
-    "<a class='anchor' href='#arguments'></a>",
+    "<", h, " class='hasAnchor' id='", make_slug(x[[1]]),"'>",
```

Comment:
:bug:

Reply:
In the future, can you please try and keep unrelated bug fixes in a separate PR? My technique is to keep a piece of scrap paper handy so I can jot down bugs to fix after I'm done with the current PR.